### ISSUE TITLE
Cultist shielded robes aren't working

### ISSUE THREAD

**Issue by shmabuu** [ParadiseSS13/Paradise#14510]
**Issue Description**:
Cult shielded suit isn't functioning as intended. May apply to other "shielded" suits, I'm unsure.

**What did you expect to happen**: 
Expected to have the taser, or baton get blocked by the shielded robes, but it didn't. Usually, the shield will protect you from any kind of stun as long as they are charged.

**What happened instead**:
Was able to stun and cuff through the shield.

**Why is this bad/What are the consequences**:
The shielded robes aren't doing their job. They are called shielded or a reason. Cultists who use them are getting caught sooner because the robes just aren't working.

**Steps to reproduce the problem**:
Wear a shielded suit such as the cultist one and get shot by a laser, taser, cuffed, or stunbatoned and it will all go through.

**When did the problem start happening**:
I noticed it in a cult round today, unsure of what broke the shielded suits.

**Comment by SteelSlayer** [comment]
Duplicate of #14466. There is a fix up: #14467

**Comment by shmabuu** [comment]
ah. didn't realize, I'll close it

**Comment by shmabuu** [comment]
PR to fix doesn't fix batons or cuffs going through the shield, reopened.

**Comment by farie82** [comment]
![image](https://user-images.githubusercontent.com/15887760/94961123-5b95b000-04f4-11eb-95b1-46134b046927.png)
Couldn't repro it
![image](https://user-images.githubusercontent.com/15887760/94961204-808a2300-04f4-11eb-87e5-2f7e1c931d9c.png)
The bubble was still up meaning the shield was still there. Was this also the case for you?

Btw cuffing was never stopped by the shield. Only attacks

**Comment by shmabuu** [comment]
Odd, I was able to stun through the shield with a baton, perhaps a visual bug? Good to know that cuffing does work though, I thought it was supposed to block it as well
